src/MainWindow/main_window.cpp

```diff
@@ -137,6 +137,17 @@ void MainWindow::Info(const ProjectInfo& info) { m_projectInfo = info; }
 
 ProjectInfo MainWindow::Info() const { return m_projectInfo; }
 
+void MainWindow::SetWindowTitle(const QString& filename, const QString& project,
```

Comment:
@KochynVolodymyr do we need this anymore ?
https://github.com/os-fpga/FOEDAG/pull/639/files#diff-9e31125b4e2b6bd449e09a6b478450cb54bf4fb9ddb23baadb06ad5d9a9bb149R63

Reply:
What do you mean?

Reply:
@zaryabkhan-rs do we need these lines ? What's the behavior when we have an empty project ?

MainWindow::MainWindow(Session* session) : m_session(session) {
  /* Window settings */
  setWindowTitle(tr("FOEDAG"));